uninstall.sh

```
@@ -0,0 +1,95 @@
+# Remove NFQUEUE rules
+for i in {1..20}; do
+    sudo iptables -D OUTPUT -m state --state NEW -j NFQUEUE --queue-num 1 2>/dev/null || break
+done
+
+# Remove BASTION_BYPASS rules
+for i in {1..20}; do
+    RULE=$(sudo iptables -S OUTPUT | grep "BASTION_BYPASS" | head -n1)
+    if [ -z "$RULE" ]; then
+        break
+    fi
+    sudo iptables -D OUTPUT $(echo $RULE | cut -d' ' -f3-) 2>/dev/null || break
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)